src/app/header/components/search-bar/search-bar.component.scss

```diff
@@ -10,6 +10,7 @@
         padding-left: 8px;
         border-radius: 8px;
         outline-style: solid;
```

Comment:
Why are you using outline instead of border here?

Reply:
Good question, I'm gonna check the difference now. I'm working on the header alignment now, it's hard to place the search in the same alignment as the two below because of the logo.

Reply:
Fixed.